lib/services/local/integrations.go

```diff
@@ -36,11 +36,25 @@ const (
 
 // IntegrationsService manages Integrations in the Backend.
 type IntegrationsService struct {
-	svc generic.Service[types.Integration]
+	svc              generic.Service[types.Integration]
+	backend          backend.Backend
+	deleteAllEnabled bool
```

Comment:
What do you think of a more general `unchecked` or `cacheMode`? The only reason why the conditional operations are guaranteed to not fail is that the external audit config is not cached, so there's no way for out of order events to cause conditional checks to fail, but it'd be cleaner to just have a flag for that - `notReferencedByEAS` could be a method on `IntegrationsService` that always returns no conditions if the flag is set, so the rest of the code can just keep using `AtomicWrite`.

Reply:
I introduced the cacheMode and deprecated the gRPC DeleteAllIntegrations